packages/relay/src/lib/types/IRequestDetails.ts

```diff
@@ -0,0 +1,4 @@
+export type IRequestDetails = {
+  requestIdPrefix: string;
+  requestIp: string;
+};
```

Comment:
Let's convert this to a class:
```typescript
export class RequestDetails {
  requestId: string;
  ipAddress: string;
  
  constructor(details?: { requestId: string, ipAddress: string }) {
    if (details) {
      this.requestId = details.requestId;
      this.ipAddress = details.ipAddress;
    }
  }
  
  get formattedRequestId(): string {
    if (!this.requestId) {
      return '';
    }
    return `[${constants.REQUEST_ID_STRING}${this.requestId}]`;
  }
}
```

Then, we can also remove the `formatRequestIdMessage` utility method and we have the flexibility of whether we want to use the raw or formatted version of the `requestId`.